### PR TITLE
Added LiveJournal

### PR DESCRIPTION
**Pull request by kojaxs**

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -93,6 +93,12 @@ websites:
       sms: Yes
       doc: https://help.linkedin.com/app/answers/detail/a_id/531/related/1
 
+    - name: Livejournal
+      url: https://livejournal.com
```
Comment (Carlgo11) [inline _data/social.yml]: The https link appears to redirect to `http://www.livejournal.com/`. I suggest you change the URL to that.

**Comment by kojaxs** [comment]
The url has been updated.

**Comment by jamcat22** [comment]
Could you please make two other quick changes?
1. Change `name: Livejournal` to `name: LiveJournal`?
2. Update the image with this more compressed version (compressed using [TinyPNG](https://tinypng.com))? 
livejournal

**Comment by kojaxs** [comment]
The name and image have been updated.

**Comment by jamcat22** [comment]
Awesome. Thanks for making those changes @kojaxs!

**Pull request merged**